lab.R

```diff
@@ -139,18 +164,23 @@ levels(bad_factor)
 
 #' To fix this, we need to set the order of the levels manually.  First, edit the next line, so that the levels correspond to the list in the documentation, with "Very Poor" as the lowest (first) value and "Very Excellent" as the highest (<http://jse.amstat.org/v19n3/decock/DataDocumentation.txt>, search for "Overall Cond").  
 
-condition_levels = c('Above_Average', 'Very_Poor', 'Below_Average')
+
+# "Above_Average" "Average"       "Below_Average" "Excellent"     "Fair"          "Good"          "Poor"          "Very_Good"     "Very_Poor"
+
+condition_levels = c("Very_Poor", "Poor", "Fair", "Below_Average", "Average", "Above_Average", "Good", "Very_Good", "Excellent")
```

Comment:
You're missing "Very_Excellent" (in the codebook but not the dataset)

Reply:
oops sorry.  I just printed our the ones in the dataset and assumed that was all of them